src/queue/bookmark/activities.ts

```diff
@@ -0,0 +1,32 @@
+import { InjectedProps } from '../common';
```

Comment:
Instead of `queue` let's name it temporal to be explicit. The directory I mean

Reply:
Cool. Let's do that.